stock_move_line_removal_date/tests/test_stock_move_line.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+from odoo import SUPERUSER_ID
+from odoo.tests import common
+
+
+class TestStockMove(common.TransactionCase):
+    def setUp(self):
+        super(TestStockMove, self).setUp()
+        self.stock_location = self.env.ref("stock.stock_location_stock")
+        self.supplier_location = self.env.ref("stock.stock_location_suppliers")
+        self.partner_delta_id = self.env.ref("base.res_partner_4")
+        self.picking_type_in = self.ref("stock.picking_type_in")
+
+        self.picking_id = self.env["stock.picking"].create(
+            {
+                "location_id": self.supplier_location.id,
+                "location_dest_id": self.stock_location.id,
+                "partner_id": self.partner_delta_id.id,
+                "picking_type_id": self.picking_type_in,
+            }
+        )
+        self.productA = self.env["product.product"].create(
+            {
+                "name": "Product A",
+                "type": "product",
+                "tracking": "serial",
+                "categ_id": self.env.ref("product.product_category_all").id,
+            }
+        )
+
+    def test_action_done(self):
+        """ This test perform to evaluate the action_done method
+        which allows to convert date to datetime with Specific hours"""
+
+        # Adding UTC timezone for SUPERUSER.
+        self.env["res.users"].browse(SUPERUSER_ID).write({"tz": "UTC"})
+
+        # Move creation
+        move1 = self.env["stock.move"].create(
+            {
+                "name": "test_in_1",
+                "location_id": self.supplier_location.id,
+                "location_dest_id": self.stock_location.id,
+                "product_id": self.productA.id,
+                "product_uom": self.productA.uom_id.id,
+                "product_uom_qty": 2.0,
+                "picking_type_id": self.env.ref("stock.picking_type_in").id,
+            }
+        )
+
+        # Move Confirmation
+        move1._action_confirm()
+        self.assertEqual(move1.state, "confirmed")
+
+        # Move assignment
```

Comment:
```suggestion
        # Confirm and assign move
        move1._action_confirm()
```
Remove unnecessary assertion.